.circle/vars.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set -e
+set -x
+
+DISTROS=(wheezy jessie trusty centos7)
+DISTRO=${DISTROS[$CIRCLE_NODE_INDEX]}
+
+# Package version & release hardcoded for now
+# TODO: Get version from `st2common` based on
+ST2PKG_VERSION=1.2dev
+# TODO: Get revision number based on existing Bintray packages (already uploaded)
+ST2PKG_RELEASE=1
+
+set +x
+echo "export DISTROS=(${DISTROS[*]})" >> ~/.circlerc
+echo "export DISTRO=${DISTRO}" >> ~/.circlerc
+echo "export ST2PKG_VERSION=${ST2PKG_VERSION}" >> ~/.circlerc
+echo "export ST2PKG_RELEASE=${ST2PKG_RELEASE}" >> ~/.circlerc
+#    - echo ST2PKG_VERSION=$(python -c 'execfile("../st2common/st2common/__init__.py"); print __version__') >> ~/.circlerc
+#    - echo "ST2PKG_RELEASE=$(scripts/bintray.sh next-revision ${DISTRO} ${ST2PKG_VERSION})" >> ~/.circlerc
+
+set -x
+cat ~/.circlerc
```

Comment:
This is totally fine: 
![kssfzu3 1](https://cloud.githubusercontent.com/assets/1533818/11410492/8b357574-93d0-11e5-9e9f-a14bc14fdbfc.png)

CircleCI env variables: https://circleci.com/docs/environment-variables and our variables combined together and the most important *sourced by default*.

The fact is that they're already combined, eg all we define in `machine:environment` already goes in that `.circlerc`.

-----

This is nice and works by default: 
```yaml
dependencies:
  override:
    - .circle/docker-compose.sh pull ${DISTRO}:
        parallel: true
  post:
    - .circle/docker-compose.sh build ${DISTRO}:
        parallel: true

test:
  override:
    - .circle/docker-compose.sh test ${DISTRO}:
        parallel: true
```

comparing to ugly way with separated file for custom variables (how I actually did before): 
```yaml
dependencies:
  override:
    - . ~/env; .circle/docker-compose.sh pull ${DISTRO}:
        parallel: true
  post:
    - . ~/env; .circle/docker-compose.sh build ${DISTRO}:
        parallel: true

test:
  override:
    - . ~/env; .circle/docker-compose.sh test ${DISTRO}:
        parallel: true
```

And don't forget that we're not only two guys who will work with this in future.
Less details and corner cases to note/remember - easier for others to use/read.